### DISCUSSION TITLE
Deactivate & Activate Triggers in Coarse Editor

### DISCUSSION THREAD

**Discussion by uf2-farmsim** [Courseplay/Courseplay_FS22#2591] [Ideas]
One thing that would take coarseplay to the next level would be additional choices in the advanced section of the editor.  If we had two more choices for waypoints.

1.  Deactivate Waypoint:  This would raise implements, turn off sprayers, raise harvester heads, etc.
2.  Activate Waypoint:  This would lower implements, turn on sprayers, lower harvester heads, etc.

If we had this, we could make custom rounded turns anywhere and fix any missed portions.  This would also make turning more realistic rather than the robotic looking turns we have now.

**Comment by Tensuko** [comment]
This is not how the complete system of tool on/off or raise and lower works.
Also the turns are based on some factors of the implements.

What do you call "robotic looking" turns.
The turns are calculcated based on radius of the tools and working witdh and so on.
If you turn on the pathfinder for turns, you may also see different turns.

Anyway, that Idea won't happen, there is no real benefit to a rather complex system that we would have to implement (more waypoint types and stuff).

**Comment by uf2-farmsim** [comment]
The turns are very unrealistic.  I work in agriculture and we only use reverse when working on the boundaries.  It adds a lot of stress to the drivetrain and makes field applications and operations very inefficient.  If reverse is needed when turning in the field, it is considered a failed turn.  The following video might help you understand how a smooth turn works.

https://youtu.be/dFifmshrmUU?t=49

For coarseplay to be more realistic, it should offer to complete sharp turns on field boundaries, then smooth rounded turns while going back and forth within the field.  Since it does not do this, I was recommending the additional tools in the coarseplay editor so users that like realism could accomplish this.

I do realise when more headlands are completed it will then make a rounded turn, but it often takes more headlands than needed to do so (when 2 should be plenty for larger implements.)

I hope this helps.

**Reply by Tensuko** [reply]
As said, try out pathfinder.
We won't add such an option to the course editor.
